src/components/Correction/CompletingParty.vue

```diff
@@ -55,11 +55,11 @@ export default class CompletingParty extends Mixins(CommonMixin) {
 
   /** The original Completing Party if found, otherwise undefined. */
   get originalCompletingParty () : OrgPersonIF {
-    if (this.isCorrectionFiling && this.getOriginalIA?.incorporationApplication) {
```

Comment:
Checking whether this is a correction filing is redundant: `this.getOriginalIA` only exists if this is a correction (and, specifically, a Ben IA correction).

Ditto in code below and in other files below.